.cirrus.yml

```diff
@@ -116,6 +116,22 @@ ext_svc_check_task:
           _gc='git config --file /root/.gitconfig'
           $_gc user.email "TMcTestFace@example.com"
           $_gc user.name "Testy McTestface"
+          # Nightly dependency-bump job: fetch latest versions of the
+          # Big Three dependencies, and run full CI test suite. Notification
+          # email will go out to monitor-list upon failure.
+          if [[ "$CIRRUS_CRON" = "treadmill" ]]; then
+              for pkg in common image/v5 storage; do
```

Comment:
Why is buildah excluded? This will fail if common makes a breaking change that also affects buildah.
IMO we need to vendor everything at once, excluding buildah will cause even more troubles.

Reply:
Not really, this thing here will just start to fail we break something in c/common. Until common is declared stable this job will fail regularly. 

Is there any reason to not just vendor everything in your treadmill PR vs this automated job?

Reply:
>Is there any reason to not just vendor everything in your treadmill PR vs this automated job?

My sanity. Buildah treadmill is already a pretty big time suck (some days). I'm reluctant to spend more time on this sort of integration work.